proposals/wac-ucr/index.bs

```diff
@@ -42,5 +42,111 @@ This is where we explain the use case.
+of a given [=identity=].
+
+An <dfn>application</dfn> is a piece of software that interfaces with a
+[=resource server=], which may operate autonomously as an
+[=authenticated agent=], or in piloted-fashion by another
+[=authenticated agent=].
+
+An <dfn>application identity</dfn> is used to uniquely identify a given
+[=application=], represented by a unique [[URI]].
+
+A <dfn>resource controller</dfn> is an [=agent=] with fully permissioned
+access and control over one or more [=resources=] residing on a
+[=resource server=] on the Web.
+
+A <dfn>resource server</dfn> is a networked system that provides an
+interface to make [=resources=] available to [=agents=] and [=applications=],
+with the ability to secure access to those [=resources=]
+through associated [=acl resources=].
+
+A <dfn>resource</dfn> is a set of data identified via a [[URI]].
+
+A <dfn>collection</dfn> is a [=resource=] that is representative of a set
+of other [=resources=], which may include other [=collections=].
+
+An <dfn>acl resource</dfn> is a [=resource=] that includes one or more
+[=authorization statements=] applied to one or more [=resources=].
+
+An <dfn>authorization statement</dfn> is an expression included in
+an [=acl resource=] that identfies permitted [=access modes=] to
+[=authorization targets=] by [=authorization subjects=].
+
+An <dfn>authorization subject</dfn> identifies the [=agent=] and/or
+[=application=] that a given [=authorization statement=] applies to.
+
+An <dfn>authorization target</dfn> identifies the resources that a given
+[=authorization statement=] applies to.
+
+An <dfn>authorization group</dfn> is a [=resource=] including
+any combination of discrete [=authorization subjects=] and
+[=authorization groups=], which can be included in an
+[=authorization statement=] as an [=authorization subject=].
+
+An <dfn>authorization condition</dfn> provides additional criteria
+to be used in the evaluation of an [=authorization statement=], beyond
+the [=authorization subject=], [=authorization target=], and [=access modes=].
+
+
+An <dfn>access mode</dfn> denotes a class of operations that an
+[=authorization subject=] can perform on an [=authorization target=] in an
+[=authorization statement=].
+
+<dfn>Read access</dfn> is an [=access mode=] that allows the
+[=authorization subject=] to the ability to read, but not modify, the
+[=authorization target=].
+
+<dfn>Write access</dfn> is an [=access mode=] that allows the
+[=authorization subject=] to the ability to create, update, or delete the
+[=authorization target=].
+
+<dfn>Append access</dfn> is an [=access mode=] that allows the
+[=authorization subject=] to add data to a resource, but not modify any
+data that already exists.
+[=authorization target=].
+
+<dfn>Control access</dfn> is an [=access mode=] that allows the
+[=authorization subject=] to view and modify [=acl resources=] associated
+with an [=authorization target=].
```

Comment:
Do these need to be defined in the UCR? If it is referring to the ACL vocab, simply refer to that.